dhall/src/Dhall/Diff.hs

```diff
@@ -541,12 +541,15 @@ skeleton (TextAppend {}) =
     <>  operator "++"
     <>  " "
     <>  ignore
-skeleton (ListLit {}) =
+skeleton (ListLit _ elems)
+    | null elems =
+            listSkeleton
```

Comment:
We could even return an `emptyListSkeleton` here that doesn't have the `...` inside.